WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -49,9 +48,18 @@
     private ApiHelper.FetchPostsTask mCurrentFetchPostsTask;
     private ApiHelper.FetchSinglePostTask mCurrentFetchSinglePostTask;
     private View mProgressFooterView;
+
+    private View mEmptyView;
+    private View mEmptyViewImage;
+    private TextView mEmptyViewTitle;
+
     private boolean mCanLoadMorePosts = true;
     private boolean mIsPage, mShouldSelectFirstPost, mIsFetchingPosts;
 
+    private enum MessageType {
```

Comment:
There are three definitions of MessageType, nearly identical. Any way we can consolidate?